### DISCUSSION TITLE
BrewTestBot and Cask Updating?

### DISCUSSION THREAD

**Discussion by iandol** [Homebrew/discussions#890] [Casks] [answered]
Hi, in this pull request: 

https://github.com/Homebrew/homebrew-cask/pull/95935

BrewTestBot used `brew bump-unversioned-casks` that incorrectly updated this cask's version number. I've made a new pull request but I was just curious if this will happen again? 

In this case, the app does have a github appcast, and I suppose it would be better if `bump-unversioned-casks` checked that before trying to extract the version from the app bundle `Info.plist`?

**Comment by Bo98** [comment, marked as answer]
Ideally the URL itself would contain the version number so that the whole `bump-unversioned-casks` wouldn't be needed.

It looks like though there is now this: https://download.zerotier.com/RELEASES/. Maybe it could be switched to use that?

**Reply by iandol** [reply]
Hi, if this would stop BrewTestBot from resetting the version number, this seems a workable solution. My current pull is here: https://github.com/Homebrew/homebrew-cask/pull/100253 — should I close that and create a new pull with the changed URL?

**Reply by Bo98** [reply]
You can also just update the current PR - whatever is easier to you. You should also change `sha256` since we can check on that now (`:no_check` was just because of the unversioned URL).

For any other questions, feel free to ask out in the PR. I'm not a part of the Cask team so I'm not entirely familiar with their processes.

**Reply by iandol** [reply]
Thank you, I modified the pull based on your advice and it got merged.
